### ISSUE TITLE
Fix rounded corner / background gap in ContextMenu

### ISSUE THREAD

**Issue by jankeromnes** [gitpod-io/gitpod#5086]
### Bug description

There is a breeze coming through this gap, and it should be fixed, ideally before the winter. 🌬️ 🥶

From https://github.com/gitpod-io/gitpod/pull/5077#discussion_r684024685 :

> **issue**: This is minor but there's a small gap on the corners due to corner radiuses. It should be barely noticable but if there's an easy fix let's also include it here. Otherwise, a follow up issue could suffice. 🏓 
> 
> | A | B |
> |-|-|
> | <img width="493" alt="Screenshot 2021-08-06 at 10 46 57 AM" src="https://user-images.githubusercontent.com/120486/128477095-bcf6f7fd-ddb5-4846-93de-3d3c2f2a38a5.png"> | <img width="482" alt="Screenshot 2021-08-06 at 10 46 43 AM" src="https://user-images.githubusercontent.com/120486/128477100-d9da63d0-94d0-4448-8c3a-de80ba9b3967.png"> |

### Steps to reproduce

1. Click on any DropDown / ContextMenu in the dashboard
2. Zoom in
3. Feelsbadman

### Expected behavior

3. Feelsgoodman

### Example repository

https://github.com/gitpod-io/gitpod

### Anything else?

Do you have any fries? 🍟

**Comment by jankeromnes** [comment]
@gtsiolis About the `good-first-issue` label: I considered adding it, but then I wasn't sure, because I myself don't know how to fix this problem (I tried a few things, like not rounding the background's corners but using `overflow: hidden` instead, but that didn't work).

In general for `good-first-issue`s, I like at least being able to point interested contributors in the right direction for a fix, but here I don't really know. 😬

**Comment by gtsiolis** [comment]
@jankeromnes fair point! No strong feelings either way. Removing the `good-first-issue` label. 🏀

**Comment by gtsiolis** [comment]
Reopening this as it's still relevant.
